settings/checkstyle/checkstyle.xml

```diff
@@ -208,7 +208,9 @@
       <property name="allowMissingReturnTag" value="true"/>
       <property name="allowedAnnotations" value="Override, Test"/>
     </module>
-    <module name="MissingJavadocMethod"/>
+    <module name="MissingJavadocMethod">
+      <property name="scope" value="protected"/>
```

Comment:
note to self: `protected` scope covers both `public` and `protected`
https://checkstyle.sourceforge.io/property_types.html#Scope